backend/src/packages/business/business.service.ts

```diff
@@ -134,15 +142,20 @@ class BusinessService implements IService {
     payload,
     businessId,
   }: DriverAddPayload): Promise<DriverAddResponseWithGroup> {
-    const doesBusinessExist = await this.findById(businessId);
+    const business = await this.findById(businessId);
 
-    if (!doesBusinessExist) {
+    if (!business) {
       throw new HttpError({
         status: HttpCode.BAD_REQUEST,
         message: HttpMessage.BUSINESS_DOES_NOT_EXIST,
       });
     }
 
+    void this.truckService.addTrucksToUser(
+      business.ownerId,
+      payload.driverTrucks,
+    );
+
```

Comment:
this table should save data on which drivers can drive on which trucks. now you save business owner id to trucks, this table is not for this purpose
first create driver, then addTrucksToDriver({ driverId: createdUser.id, possibleTruckIds: payload.driverTrucks });